### ISSUE TITLE
Login/Create account pages improvment

### ISSUE THREAD

**Issue by ravinderjangra** [maidsafe/safe-authenticator-mobile#207]
Need to scroll to see & click on the login/create button on smaller screens, can this be auto brought into full view on focus? https://safenetforum.org/t/new-release-safe-authenticator-v0-2-0-safe-mobile-browser-v0-3-0/31092/34?u=stephenc

https://safenetforum.org/t/new-release-safe-authenticator-v0-2-0-safe-mobile-browser-v0-3-0/31092/36?u=stephenc

**Comment by ravinderjangra** [comment]
@S-Coyle I am not 100% clear what's the improvement we want to add.

**Comment by S-Coyle** [comment]
@ravinderjangra My understanding is that the issue with the scroll on login/create was sorted in the patch fix, so linked comments on that can be ignored now.

I believe the only outstanding issue from the linked comments was the suggestion that there could maybe be some simple way of knowing while in the browser whether it was authenticated or not, e.g. a `state` icon was suggested. I've logged [this](https://github.com/maidsafe/safe-mobile-browser/issues/239) in the mobile Browser repo to track.
Closing issue.
